doc/user/acl.md

```diff
@@ -0,0 +1,150 @@
+# DAOS Access Control
+
+Client access to DAOS resources like pools and containers is controlled by
+Access Control Lists (ACLs). A DAOS ACL is a list of zero or more Access Control
+Entries (ACEs). ACEs are the individual rules applied to each access decision.
+
+## Access Control Entries
+
+In the input and output of DAOS tools, an ACE is defined using a colon-separated
+string format:\
+`TYPE:FLAGS:IDENTITY:PERMISSIONS`
```

Comment:
Suggestion: replace identity with principal globally in the documentation.

Reply:
Good idea - this was pretty inconsistent. Fixing.